### DISCUSSION TITLE
How to define a global CSS?

### DISCUSSION THREAD

**Discussion by compscibro** [thuliteio/doks#1105] [Q&A] [answered]
Hi, I have tables in the documentation that I am working on. Currently, I am able to apply my own design for the tables by adding inline html and css in the markdown files.

However, if I want to apply the same design for all tables in all of the markdown files, I may need to define a global CSS. What's your take on this? Thanks!

**Comment by h-enk** [comment, marked as answer]
Know that you can add a class to a table in markdown by adding e.g. `{.table .table-hover}` (see also [Tables](https://getbootstrap.com/docs/5.3/content/tables/))

```md
| Syntax | Description |
| --- | ----------- |
| Header | Title |
| Paragraph | Text |
{.table .table-hover}
```

Add global custom table styling in `themes/my-doks-theme/assets/scss/common/_custom.scss` (see also https://github.com/gethyas/doks/discussions/1107#discussioncomment-7212876)

**Comment by compscibro** [comment]
Thanks!
